rancid-tomatillos/src/Components/Card/Card.js

```diff
@@ -1,23 +1,17 @@
 import './Card.css';
-import React, {Component} from "react";
+import React from "react";
+
+const Card = ({key, poster, rating, title}) => {
```

Comment:
So glad we refactored this component. Turns out we didn't need any state after all... ✨